src/components/Input/Input.stories.tsx

```diff
@@ -33,6 +33,7 @@ export default {
       'onVisibleChange',
       'clearFormValueOnUnmount',
       'name',
+      'size',
```

Comment:
We should use including instead of excluding

Reply:
We could forget to add a prop when it's been added to the component. But in general, agree include is better than exclude